test/integration/included/stanc.expected

```diff
@@ -1,41 +1,47 @@
   $ ../../../../install/default/bin/stanc --include_paths="../included" --auto-format badrecurse1.stan
 
 Syntax error at file ../included/badrecurse1.stan, line 1, column 0, included from
-file ../included/badrecurse1.stan, line 1, column 25, included from
-file badrecurse1.stan, line 1, column 25, includes error:
+file ../included/badrecurse1.stan, line 1, column 0, included from
+file ../included/badrecurse1.stan, line 1, column 0, included from
+file badrecurse1.stan, line 1, column 0, includes error:
    -------------------------------------------------
      1:  #include badrecurse1.stan
          ^
    -------------------------------------------------
 
 Found cyclical include structure.
 
+
   $ ../../../../install/default/bin/stanc --include_paths="../included" --auto-format badrecurse2.stan
 
-Syntax error at file ../included/badrecurse3.stan, line 1, column 0, included from
-file ../included/badrecurse2.stan, line 1, column 25, included from
-file ../included/badrecurse3.stan, line 1, column 25, included from
-file badrecurse2.stan, line 1, column 25, includes error:
+Syntax error at file ../included/badrecurse2.stan, line 1, column 0, included from
+file ../included/badrecurse3.stan, line 1, column 0, included from
+file ../included/badrecurse2.stan, line 1, column 0, included from
+file ../included/badrecurse3.stan, line 1, column 0, included from
+file badrecurse2.stan, line 1, column 0, includes error:
    -------------------------------------------------
-     1:  #include badrecurse2.stan
+     1:  #include badrecurse3.stan
          ^
    -------------------------------------------------
 
 Found cyclical include structure.
```

Comment:
Similarly, this would be `'badrecurse3.stan' included itself.`

Reply:
Done.